src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    Animal.alive.push(this);
```

Comment:
```suggestion
    Animal.alive.push(this);
```
Why do you adding only herbivores or carnivores to `alive` array? Don't you think that it will be better to put this line `Animal.alive.push(this);` to the `Animal` class and two other classes will inherit it?